components/edit_post/edit_post.tsx

```diff
@@ -5,6 +5,8 @@ import React, {ClipboardEventHandler, memo, useCallback, useEffect, useRef, useS
 import classNames from 'classnames';
 import {useIntl} from 'react-intl';
 
+import {ClientConfig} from 'mattermost-redux/types/config';
+
```

Comment:
```suggestion

```
nit : both above and below are ts types

Reply:
re-ordered imports a bit to better structure them